catalog/plugin/catalog-plugin-expirationdate/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Expiration Date Pre-Ingest Plugin"
+         id="org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin"
+         description="Catalog pre-ingest plugin to set an expiration date on metacards">
+
+        <AD
+                description="If this is checked, overwrite all 'blank' expiration dates in metacards. If this is not checked, leave metacards with 'blank' expiration dates as-is."
+                name="Overwrite If Empty" id="overwriteIfBlank"
+                required="true" type="Boolean"
+                default="true"/>
```

Comment:
If the bundle is installed by default, we should probably default this to false which will force an admin to turn this capability on and configure the expiration time appropriately.